### ISSUE TITLE
UILineRenderer's start position v.s. anchor

### ISSUE THREAD

**Issue by SimonDarksideJ** [Unity-UI-Extensions/com.unity.uiextensions#174]
<sup>Issue created by 曾俊龍（阿龍哥） as [Bitbucket Issue #&#8203;138](https://bitbucket.org/UnityUIExtensions/unity-ui-extensions/issues/138) on 2017.04.24 12:56.</sup>
The anchor point is not affect how the UILineRenderer drawing. 
Each point in points, should be related to the anchor point instead of anchor(0, 0).

**Comment by SimonDarksideJ** [comment]
<sup>On [2017.04.24 13:35](https://bitbucket.org/UnityUIExtensions/unity-ui-extensions/issues/138#comment-36168590), 曾俊龍（阿龍哥） commented:</sup>
by the way, to fix the issue, just rewrite the line 134 & 135
from
```c#
    var offsetX = -rectTransform.pivot.x * rectTransform.rect.width;
    var offsetY = -rectTransform.pivot.y * rectTransform.rect.height;

```
to

```c#
    var offsetX = 0f;
    var offsetY = 0f;
```

And also there is a bug that sizeX & sizeY would be negative when !relativeSize and UseMargins.
In fact, I think the UseMargins is meaningless when the points are more than 2, I suggest removing it.

**Comment by SimonDarksideJ** [comment]
<sup>On [2017.05.08 00:16](https://bitbucket.org/UnityUIExtensions/unity-ui-extensions/issues/138#comment-36632110), @SimonDarksideJ commented:</sup>
Removed margins in the end and found that it works properly if you use :
```
    var offsetX = -rectTransform.pivot.x * sizeX;
    var offsetY = -rectTransform.pivot.y * sizeY;
```
Also rewrote the Size lines to work better.

**Comment by SimonDarksideJ** [comment]
<sup>On [2017.05.08 00:18](https://bitbucket.org/UnityUIExtensions/unity-ui-extensions/issues/138#comment-36632117), @SimonDarksideJ commented:</sup>
Resolves #138

Line Renderer now renders based on Pivot and resolves drawing issue with Editor script when using !Relative.

However, need to update editor script to work with relative drawing.


Referenced by 1482d7c44057

**Comment by SimonDarksideJ** [comment]
<sup>On 2017.05.08 00:18 @SimonDarksideJ modified issue:</sup>
**status** changed `new` → `resolved`
